configurations/pipeline_config.json

```diff
@@ -136,6 +136,8 @@
     {"RapidProKey": "Young_People (Run ID) - mag_s01_w10_follow_up", "PipelineKey": "young_people_issues_run_id"}
   ],
   "PipelineName": "all_quarters",
+  "MoveWSMessages": false,
+  "AnalysisFilesMode": false,
```

Comment:
Can we call this e.g. 'GenerateAnalysisFiles'? It's not immediately obvious what 'AnalysisFilesMode' is and what true means